src/coreclr/src/vm/jitinterface.cpp

```diff
@@ -7243,6 +7243,39 @@ bool getILIntrinsicImplementationForUnsafe(MethodDesc * ftn,
     return false;
 }
 
+bool getILIntrinsicImplementationForMemoryMarshal(MethodDesc * ftn,
```

Comment:
Could you please the intrinsic expansion to `src\coreclr\src\tools\Common\TypeSystem\IL\Stubs` as well?